pkgs/tools/system/sleuthkit/default.nix

```diff
@@ -1,9 +1,47 @@
-{ lib, stdenv, fetchFromGitHub, autoreconfHook, libewf, afflib, openssl, zlib }:
+{ lib, stdenv, fetchFromGitHub, autoreconfHook, libewf, afflib, openssl, zlib, openjdk, perl, ant }:
 
 stdenv.mkDerivation rec {
   version = "4.11.0";
   pname = "sleuthkit";
 
+  rdeps = stdenv.mkDerivation rec {
+    # Fetch libraries using a fixed output derivation
+
+    version = "1.0";
+    pname = "sleuthkit-deps";
+    nativeBuildInputs = [ autoreconfHook openjdk ant ];
+
+    src = fetchFromGitHub {
+        owner = "sleuthkit";
+        repo = "sleuthkit";
+        rev = "${pname}-${version}";
+        sha256 = "sha256:1dh06k8grrj3wcx3h9m490p69bw41dz6cv8j5j1drpldmv67k3ki";
+    };
+
+    IVY_HOME = "/build/.ant";
+    phases = "unpackPhase buildPhase installPhase";
+
+    buildPhase = ''
+        pushd bindings/java
+        ant retrieve-deps
+        popd
+        pushd case-uco/java
+        ant get-ivy-dependencies
+        popd
+    '';
+
+    installPhase = ''
+        mkdir -m 755 -p $out/bindings/java
+        cp -r bindings/java/lib $out/bindings/java
+        mkdir -m 755 -p $out/case-uco/java
+        cp -r case-uco/java/lib $out/case-uco/java
+        cp -r $IVY_HOME/lib $out
+        chmod -R 755 $out/lib
+    '';
+    outputHashMode = "recursive";
+    outputHash = "sha256:0fq7v6zlgybg4v6k9wqjlk4gaqgjrpihbnr182vaqriihflav2s8";
```

Comment:
```suggestion
    outputHash = "0fq7v6zlgybg4v6k9wqjlk4gaqgjrpihbnr182vaqriihflav2s8";
```

Reply:
Do we have a written down policy on hash format anywhere? I am under an impression that format-prefixed hashes are the long-term happy future…

Reply:
> format-prefixed hashes are the long-term happy future…

You mean sri hashes? They start with sha256-, not sha256:.

Reply:
That depends on the encoding base, apprently, and right now Nix itself prefers printing hashes in `sha256:` prefixed format.

Reply:
Yeah and we never prefix hashes with sha256: at any place despite nix printing it like this.

Reply:
I see quite a few packages where we do, as we should as long as that's the stable Nix default for output.

Reply:
Then they should be converted like it was done several times in the past in treewide commits.

Reply:
Well, there yeah, but the real question is what's the point of removing the prefix time and time again instead of waiting until stable Nix stops recommeding the encoding you are removing.

Reply:
its prefix-less, then. Done.

Reply:
I had to 
   outputHashAlgo = "sha256";
otherwise
   error: hash '0fq7v6zlgybg4v6k9wqjlk4gaqgjrpihbnr182vaqriihflav2s8' does not include a type
So is the guideline here that outputHashAlgo is preferred instead of a prefix?

Reply:
I am not sure we have guidelines (that would be a written part og the manual), but similar code paths like fetching Cargo dependencies pass `outputHashAlgo = "sha256"` most of the time.